framework/src/transfers/MultiAppCopyTransfer.C

```diff
@@ -29,19 +30,19 @@ template<>
 InputParameters validParams<MultiAppCopyTransfer>()
 {
   InputParameters params = validParams<MultiAppTransfer>();
-  params.addRequiredParam<AuxVariableName>("variable", "The auxiliary variable to store the transferred values in.");
-  params.addRequiredParam<VariableName>("source_variable", "The variable to transfer from.");
-
+  params.addDeprecatedParam<AuxVariableName>("variable", "The auxiliary variable to store the transferred values in.", "This is being replaced by 'to_variable'.");
+  params.addDeprecatedParam<VariableName>("source_variable", "The variable to transfer from.", "This is replaced by 'from_variable'.");
+  params.addRequiredParam<VariableName>("to_variable", "The variable that the solution is being transfered into.");
+  params.addRequiredParam<VariableName>("from_variable", "The variable to transfer from.");
```

Comment:
Fine - but let's address that in another PR and change them all at once if you feel that strongly about it. Right now they are all using this naming scheme (not mine).